src/server/services/EmailService.ts

```diff
@@ -51,4 +52,27 @@ export default class EmailService {
         },
       });
   }
+
+  /**
+   * Sends an email with a link to confirm the user's account.
+   * @param request The web request associated with the email.
+   * @param account The account associated with the request.
+   * @param event The event associated with the request.
+   */
+  async sendEventAcceptanceEmail(request: Request, admin: Admin, event: TESCEvent, userEmail: string) {
+    Logger.info(`Sending acceptance email to ${userEmail} for event ${event.alias} by ${admin.username}`);
+
+    const ACCEPTANCE_EMAIL_TEMPLATE_ID = Config.SendGrid.AcceptanceEmailID
+
+    const msg = {
+      to: userEmail,
+      from: 'no-reply@tesc.events',
+      templateId: ACCEPTANCE_EMAIL_TEMPLATE_ID,
+      dynamic_template_data: {
+        event,
+      }
+    };
+
+    return sendAcceptanceEmail(msg)
```

Comment:
`sendAcceptanceEmail` just calls `sendgrid.send(msg)`.
We can get rid of `~/Config/Mailer.ts` if we don't need to set up templates (as well as the email views folder and ENV variables).

Reply:
reworked